posixlib/src/main/scala/scala/scalanative/posix/signal.scala

```diff
@@ -0,0 +1,411 @@
+package scala.scalanative
+package posix
+
+import scala.scalanative.unsafe._
+
+/**
+ * Some of the functionality described on this reference page extends the ISO C standard.
+ * Applications shall define the appropriate feature test macro (see XSH The Compilation Environment)
+ * to enable the visibility of these symbols in this header.
+ *
+ * Use clib macros and functions to get the full POSIX interface
+ *
+ * Note 1: The functionality described may be removed in a future version of this volume of POSIX.1-2017
+ * Note 2: The functionality is an option marked as XSI (X/Open System Interfaces)
+ *
+ * Note 1 or 2: followed by Linux or macOS means that the feature is not available on that platform with
+ * default compile options so the code is commented out. The code is commented out so the next developer
+ * that comes along will not have to rediscover these limitations.
+ */
+@extern
+object signal {
+  // define the following macros, which shall expand to constant expressions with distinct values
+  // that have a type compatible with the second argument to, and the return value of, the signal() function,
+  // and whose values shall compare unequal to the address of any declarable function.
+
+  // Note 1: Linux
```

Comment:
Having the clue is good, but I must confess that I can not de-code it.
Is it trying to say that the XSI obsolete SIG_HOLD is defined 
only on Linux? or not on Linux?  

As someone who has spent __lots__ of time in understandably
incomplete SN posix header files I appreciate having the 
comment marker for entry points in the standard. They let
me know that the entry point was considered and is 
intentionally not implemented.

The same comment convention is used in other places.

Reply:
See this - was updated.
https://github.com/scala-native/scala-native/pull/1362/files/54a95930e67d8824aa17a5375fc3e7244d29cb79#diff-dc2e6a694318d6132b77e0d77932935258be9cda1c6c2d929cd50e8e96054f8eR16-R18